themes/adminimal_medela/build/js/publication-rules.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @file
+ * Custom functionality for publication rules.
+ */
+(function ($, Drupal, window) {
+
+  'use strict';
+
+  Drupal.behaviors.adminimalMedelaPublicationRules = {
+    attach: function (context, settings) {
```

Comment:
Consider adding some code comments describing what this code does/why it is here.